### PR TITLE
Fix mistake in RSpec/PredicateMatcher doc

### PR DESCRIPTION
**Pull request by pirj**
Found in #919

---

Before submitting the PR make sure the following are checked:

* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [-] Added tests.
* [x] Updated documentation.
* [-] Added an entry to the `CHANGELOG.md` if the new code introduces user-observable changes.
* [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

File: lib/rubocop/cop/rspec/predicate_matcher.rb [hunk 1]
```diff
@@ -277,7 +277,7 @@ def replacement_matcher(node)
       #   expect(foo).to be_something
```
Comment (Darhazer) [inline lib/rubocop/cop/rspec/predicate_matcher.rb:277]: Actually this one should be updated as well

Comment (pirj) [reply lib/rubocop/cop/rspec/predicate_matcher.rb:277]: To what exactly?

Comment (Darhazer) [reply lib/rubocop/cop/rspec/predicate_matcher.rb:277]: Oh, sorry, misread the examples :facepalm: 
Seems not a good day for me to do reviews :)

Comment (Darhazer) [reply lib/rubocop/cop/rspec/predicate_matcher.rb:277]: What about line 285?

Comment (pirj) [reply lib/rubocop/cop/rspec/predicate_matcher.rb:277]: Good catch, missed that other one!

**Pull request merged**